lib/ember_cli/build_monitor.rb

```diff
@@ -32,7 +32,7 @@ def wait!
     attr_reader :name, :paths
 
     def complete?
-      !paths.lockfile.exist?
+      !paths.lockfile.exist? && paths.dist.join("index.html").exist?
```

Comment:
Does this fix the bug you've been seeing?

The addon creates the lockfile when the build starts, then removes it when it's finished.

If the lockfile no longer exists, I'm not sure what good waiting on the `index.html` file will do.

It's likely that it'll hang infinitely